src/main/java/com/tutti/server/core/payment/payload/response/TossPaymentsCancelResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.tutti.server.core.payment.payload.response;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+public record TossPaymentsCancelResponse(
+        String status,
+        List<CancelDetail> cancels
```

Comment:
요 DTO의 용도가 궁금합니당
왜 List<CancelDetail> cancels 리스트를 받는지?

Reply:
토스에서 응답이 올 때 canceldAt 값이 중첩 json안에 있어서 저렇게 작성했어여
from메서드랑 builder를 안쓰고 DTO랑 바로 매핑시키려면 구조를 맞춰야한다고해서 저렇게 작성했습니다!

Reply:
그렇군요
그러면 값이 List 여서가 아니고, depth 때문이라면 이렇게 해도 되지 않나용??
```suggestion
        CancelDetail cancel
```

Reply:
토스에서 응답을 배열 형태로 줘서 저희는 부분취소가 없긴해도 토스에서 주는 응답형태를 그대로 따라야할 것 같아요